samples/subsys/usb/cdc_acm/prj.conf

```diff
@@ -6,7 +6,7 @@ CONFIG_USB=y
 CONFIG_USB_DEVICE_STACK=y
 CONFIG_SYS_LOG_USB_DRIVER_LEVEL=1
 CONFIG_USB_CDC_ACM=y
-CONFIG_SYS_LOG_USB_LEVEL=1
+CONFIG_SYS_LOG_USB_DEVICE_LEVEL=1
```

Comment:
Please move a line up, then it looks more consistent

Reply:
Ok, I will put it next to SYS_LOG_USB_DRIVER_LEVEL